op-node/rollup/derive/l1_block_info.go

```diff
@@ -210,19 +210,19 @@ func (info *L1BlockInfo) unmarshalBinaryEcotone(data []byte) error {
 		return err
 	}
 	if err := binary.Read(r, binary.BigEndian, &info.BaseFeeScalar); err != nil {
-		return fmt.Errorf("invalid ecotone l1 block info format")
```

Comment:
This is a good change but would be better if there was a sentinel error that was returned in all cases

Reply:
Nice advice!